pages/collectivites/_collectiviteId/procedures/add.vue

```diff
@@ -30,11 +29,12 @@ export default {
       icons: { mdiChevronLeft }
     }
   },
-  async mounted () {
-    this.collectivite = (await axios({
-      url: `/api/geo/${this.$route.params.collectiviteId.length > 5 ? 'intercommunalites' : 'communes'}/${this.$route.params.collectiviteId}`,
-      method: 'get'
-    })).data
+  mounted () {
+    fetch(`https://nuxt3.docurba.incubateur.net/api/geo/search/collectivites?code=${this.$route.params.collectiviteId}`)
+      .then(async (res) => {
```

Comment:
Je comprends le passage de `axios` à `fetch` mais pourquoi le passage de `async/await` à `.then()` ?

J'ai une préférence pour l'utilisation de `async/await` donc ce changement m'embête, sauf s'il améliore un comportement.

Reply:
C'est pas une bonne practice de faire systématiquement de l'await sur des calls qui sont pas bloquant. Si on veut systématiquement faire de l'async await faut que je bouge ça dans une methode pour pas avoir a await la methode dans le mounted.

Reply:
Mais ici, nous avons besoin de `await` puisque les lignes 39 à 46 utilisent `this.collectivite`.

Il faut retourner à `async/await` car en l'état, la PR casse la création de procédure avec l'erreur suivante :
```
TypeError: null is not an object (evaluating 'this.collectivite.departementCode')
```